charts/stable/valheim/questions.yaml

```diff
@@ -16,92 +16,74 @@ questions:
       type: dict
       attrs:
         - variable: SUPERVISOR_HTTP_USER
-          label: SUPERVISOR_HTTP_USER
+          label: Supervisor User
           schema:
             type: string
-            default: admin
+            default: ''
             required: true
         - variable: SUPERVISOR_HTTP_PASS
-          label: SUPERVISOR_HTTP_PASS
+          label: Supervisor PASS
           schema:
             type: string
             required: true
             private: true
-            default: REPLACETHIS
+            default: ''
         - variable: SERVER_PASS
-          label: SERVER_PASS
+          label: Server Pass
           schema:
             type: string
             required: true
             private: true
-            default: REPLACETHIS
+            default: ''
   - variable: env
     group: App Configuration
     label: Image Environment
     schema:
       additional_attrs: true
       type: dict
       attrs:
-        - variable: STATUS_HTTP
-          label: STATUS_HTTP
-          description: Turn on the status http server. Only useful on public servers (SERVER_PUBLIC=true).
-          schema:
-            type: boolean
-            default: true
-        - variable: SUPERVISOR_HTTP
-          label: SUPERVISOR_HTTP
-          description: Turn on supervisor's http server.
-          schema:
-            type: boolean
-            default: true
         - variable: SERVER_NAME
-          label: SERVER_NAME
+          label: Server Name
           description: Name that will be shown in the server browser.
           schema:
             type: string
             default: My Server
             required: true
         - variable: WORLD_NAME
-          label: WORLD_NAME
+          label: World Name
           description: Name of the world without .db/.fwl file extension.
           schema:
             type: string
             default: Dedicated
             required: true
         - variable: SERVER_PUBLIC
-          label: SERVER_PUBLIC
-          description: Whether the server should be listed in the server browser (true) or not (false).
+          label: Public Server
+          description: Whether the server should be listed in the server browser.
```

Comment:
Curious about this change too -- I took it straight from the Dockerfile author's repo.  Seems like unnecessary editorializing to change it here.

Reply:
In this case, I think it's redundant to  say that it's true or false, given that is a boolean field.

Reply:
While I agree, it's present in two different description elements.  It seems like a bit of a "slippery slope" when representing the behavior of the system this simply presents access to instead of effecting that system itself.

https://github.com/lloesche/valheim-server-docker#environment-variables